.github/workflows/android.yml

```diff
@@ -35,10 +35,13 @@ jobs:
       endsWith(github.ref, '/next') &&
       !contains(github.event.head_commit.message, 'release prod')
     steps:
+      - name: Pull dev vercel secrets
+        if: ${{ !endsWith(github.ref, '/next') }}
+        run: vercel pull --yes --environment=preview --git-branch=${{ github.ref_name }} --token=${{ secrets.VERCEL_TOKEN }}
       - name: Pre-checks - Env is Dev
         run: |
           echo "ENV=development" >> $GITHUB_ENV
-          echo "SECRET_NAME=DEV_ENV" >> $GITHUB_ENV
+          cat .vercel/.env.preview.local >> $GITHUB_ENV
           echo "APPCENTER_NAME=GoodDollar/GoodDollar-Android-development" >> $GITHUB_ENV
           echo "APPCENTER_TOKEN=${{ secrets.APPCENTER_ANDROID_DEV }}" >> $GITHUB_ENV
           echo "APPCENTER_CODEPUSH_TOKEN=${{ secrets.APPCENTER_CODEPUSH_DEV }}" >> $GITHUB_ENV
```

Comment:
why not move these to vercel also?

Reply:
See original issue, We could, I just don't have the values for them to do it currently
https://github.com/GoodDollar/GoodDAPP/issues/3992#issuecomment-1566689576

Reply:
ask @johnsmith-gooddollar if he has the tokens if not
you have access to appcenter, you should be able create new tokens

Reply:
@L03TJ3 please create the new ones